shacl12-core/index.html

```diff
@@ -3012,9 +3055,16 @@ <h5>Conforms (sh:conforms)</h5>
 						<p>
 							Each SHACL instance of <code>sh:ValidationReport</code> in the results graph has exactly one value for the property <code>sh:conforms</code> and the value is of datatype <code>xsd:boolean</code>.
 							It represents the outcome of the <a>conformance checking</a>.
-							The value of <code>sh:conforms</code> is <code>true</code> if and only if the <a>validation</a> did not produce any <a>validation results</a> with a severity level representing a violation, and <code>false</code> otherwise.
 						</p>
 					</section>
+          <section id="conformanceDisallows">
+            <h5>Conformance Disallow Set (sh:conformanceDisallows)</h5>
+            <p>
+              Each SHACL instance of <code>sh:ValidationReport</code> in the results graph can have one or more values
+              for the property <code>sh:conformanceDisallows.</code> All values combined define the set of disallowed
+              severity levels. The values must be valid severity levels.
```

Comment:
What does the last sentence entail - how can a severity level be non valid? I think you can delete the sentence and just say they need to be IRIs, because anyone can invent their own levels, can't they?